src/joystick/hidapi/SDL_hidapi_gamecube.c

```diff
@@ -195,6 +203,7 @@ HIDAPI_DriverGameCube_InitDevice(SDL_HIDAPI_Device *device)
         }
     }
 
+    SDL_AddHintCallback(SDL_HINT_JOYSTICK_GAMECUBE_RUMBLE_BRAKE, SDL_JoystickGameCubeRumbleBrakeHintChanged, NULL);
```

Comment:
I think you also need a call to remove the hint callback on quit.

Reply:
Also, this initialization function is called for each device context, so you should pass in ctx as the userdata and then move useRumbleBrake into the context structure.

Reply:
Done, and I also added the Del call

Reply:
Whoops, didn't catch the second comment, I'm adding that change now

Reply:
Alright, that's done too